changelog/v1.3.2/fix-install-with-glooe-chart-override.yaml

```
@@ -0,0 +1,7 @@
+changelog:
+  - type: FIX
+    description: fix `glooctl install ingress/knative` to nest helm overrides under "gloo" when a GlooE chart is passed with `-f` flag
+    issueLink: https://github.com/solo-io/gloo/issues/1999
+  - type: FIX
+    description: fix `glooctl install gateway enterprise` to not set `installConfig.Enterprise = true` when an open-source chart is passed with `-f` flag
```

can we update this to be more user-friendly? perhaps
```suggestion
    description: fix `glooctl install gateway enterprise` to stop nesting helm overrides under "gloo" when a Gloo open-source chart is passed with the `-f` flag
```

---

I'm happy to change this but would love to understand why this is better and how we write changelogs- does it have to do with understanding the change without knowing the implementation?